### ISSUE TITLE
Getting ip address for docker0 fails

### ISSUE THREAD

**Issue by heppu** [datawire/telepresence#202]
### What were you trying to do?

Run telepresence with container mode.

### What did you expect to happen?

Telepresence to get ip address for docker0 interface.

### What happened instead?

Telepresence crashes because it can't parse ip address. Newer versions of ifconfig have `inet` instead of `inet addr` in output so regex fails.

### Automatically included information

Command line: `['/usr/local/bin/telepresence', '-m', 'container', '--new-deployment', 'example', '--docker-run', '-i', '-t', 'golang:1.8', '/bin/bash']`
Version: `0.52`
Python version: `3.6.1 (default, Mar 27 2017, 00:27:06) 
[GCC 6.3.1 20170306]`
kubectl version: `(error: Command '['kubectl', 'version', '--short', '--client']' returned non-zero exit status 1.)`
oc version: `(error: [Errno 2] No such file or directory: 'oc')`
OS: `Linux FL013 4.11.6-3-ARCH #1 SMP PREEMPT Thu Jun 22 12:21:46 CEST 2017 x86_64 GNU/Linux`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence", line 106, in call_f
    return f(*args, **kwargs)
  File "/usr/local/bin/telepresence", line 1708, in go
    runner, args
  File "/usr/local/bin/telepresence", line 1052, in start_proxy
    processes, socks_port, ssh = connect(runner, remote_info, args)
  File "/usr/local/bin/telepresence", line 809, in connect
    )[0]
IndexError: list index out of range

```

Logs:

```
Checking {'pod-template-hash': '2457151307', 'telepresence': 'fec69198-8e23-4ee2-99c3-a00766b84e88'} (phase Running)...
Looks like we've found our pod!
Running: (['kubectl', '--context', 'default-system', '--namespace', 'default', 'get', 'pod', 'example-2457151307-7ghsd', '-o', 'json'],)... ran!
Running: (['kubectl', '--context', 'default-system', '--namespace', 'default', 'port-forward', 'example-2457151307-7ghsd', '46179:8022'],)...Running: (['ifconfig', 'docker0'],)... ran!

```

**Comment by itamarst** [comment]
Hi, sorry you had problems, and thanks for reporting it. Will check out PR next.

**Comment by itamarst** [comment]
Hoping to get this merged and released today.

We'd like to send you a t-shirt as a thank you for the patch - email me at itamar@datawire.io with your mailing address and t-shirt size if you're interested.

**Comment by heppu** [comment]
Thanks will do!

**Comment by itamarst** [comment]
I released the fix, just upgrade to v0.54 and it should work.
